FlowCrypt/Controllers/Settings/Settings List/SettingsViewController.swift

```diff
@@ -116,14 +116,12 @@ extension SettingsViewController {
 
         switch setting {
         case .keys:
-            showSpinner()
             do {
                 viewController = try await KeySettingsViewController(appContext: appContext)
             } catch {
                 viewController = nil
                 showAlert(message: error.localizedDescription)
             }
-            hideSpinner()
```

Comment:
are these spinners not needed anymore?

Reply:
Well, spinner usually appears and disappears in 0.3-0.5 seconds and someone might think it's UI tweak issue.
Let me add spinner if you want so.

Reply:
Ah, in this case no need to add spinner here. I thought maybe it's related to some delays which appeared with SSL pinning